### PR TITLE
add docker-compose setup

### PR DESCRIPTION
**Pull request by brianpeiris**

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -62,7 +63,7 @@ config :ret, Ret.Repo,
   username: "postgres",
   password: "postgres",
   database: "ret_dev",
-  hostname: "localhost",
+  hostname: "db",
```
Comment (brianpeiris) [inline config/dev.exs]: Not sure if there is a better way to configure the database host name specifically for docker-compose.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM elixir:1.5
```
Comment (gfodor) [inline Dockerfile]: FYI 1.6 is out

File: config/dev.exs (hunk 1 above)
Comment (gfodor) [reply config/dev.exs]: hmm yeah i am not sure. on the fence which one makes more sense.

File: Dockerfile (hunk 2 above)
Comment (brianpeiris) [reply Dockerfile]: I stuck to the version used in plan.sh, assuming that's what we actually use https://github.com/mozilla/reticulum/blob/master/habitat/plan.sh#L21

Comment (gfodor) [reply Dockerfile]: ah yes, waiting on habitat to upgrade

**Comment by gfodor** [comment]
lgtm. i might try to fix up the host thing if it ends up being annoying

**Comment by brianpeiris** [comment]
cool, thanks

**Pull request merged**